wurst/objects/items/BootsDefinition.wurst

```diff
@@ -0,0 +1,277 @@
+    getAbilityDataField(ABILITY_INT_BONUS_6,     "DataB1").color(COLOR_TURQUOISE),
+    getAbilityDataField(ABILITY_ARMOR_BONUS_3,   "DataA1").color(COLOR_GREEN)
+) + TT_TRANSMUTE_SUFFIX
+
+public let TT_STEEL_BOOTS = TT_UNFORMAT_ARMOR_BOOT.format(
+    getAbilityDataField(ABILITY_SPEED_BONUS_100, "DataA1").color(COLOR_ORANGE),
+    getAbilityDataField(ABILITY_INT_BONUS_10,     "DataB1").color(COLOR_TURQUOISE),
+    getAbilityDataField(ABILITY_ARMOR_BONUS_5,   "DataA1").color(COLOR_GREEN)
+) + TT_TRANSMUTE_SUFFIX
+
+public let TT_ANA_BOOTS = TT_UNFORMAT_ANA_BOOT_STRING.format(
+    getAbilityDataField(ABILITY_SPEED_BONUS_200, "DataA1").color(COLOR_ORANGE),
+    getAbilityDataField(ABILITY_ARMOR_BONUS_5,   "DataA1").color(COLOR_GREEN),
+    4..toToolTipGreen(),
+    EVASION_CHANCE.toToolTipPurple()
+)
+
+public let TT_HYDRA_FINS = TT_UNFORMAT_HYDRA_FINS_STRING.format(
+    getAbilityDataField(ABILITY_SPEED_BONUS_100, "DataA1").color(COLOR_ORANGE),
+    getAbilityDataField(ABILITY_ARMOR_BONUS_5,   "DataA1").color(COLOR_GREEN),
+    5..toToolTipGreen(),
+    getAbilityDataField(ABILITY_SPELL_SHIELD, "Cool1").color(COLOR_LIGHT_BLUE)
+)
+
+@compiletime function createSpeedBonusAbility()
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_60)
+        ..setMovementSpeedBonus(1, 60)
+        ..setName("Movement Speed Bonus 60")
+        ..setEditorSuffix("(Wurst)")
+
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_70)
+        ..setMovementSpeedBonus(1, 70)
+        ..setName("Movement Speed Bonus 70")
+        ..setEditorSuffix("(Wurst)")
+
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_80)
+        ..setMovementSpeedBonus(1, 80)
+        ..setName("Movement Speed Bonus 80")
+        ..setEditorSuffix("(Wurst)")
+
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_100)
+        ..setMovementSpeedBonus(1, 100)
+        ..setName("Movement Speed Bonus 100")
+        ..setEditorSuffix("(Wurst)")
+
+    new AbilityDefinitionMoveSpeedBonus(ABILITY_SPEED_BONUS_200)
+        ..setMovementSpeedBonus(1, 200)
+        ..setArtTarget(Abilities.faerieDragonMissile)
+        ..setTargetAttachmentPoint("foot,left")
+        ..setTargetAttachmentPoint1("foot,right")
+        ..setTargetAttachments(2)
+        ..setName("Movement Speed Bonus 200")
+        ..setEditorSuffix("(Wurst)")
+
+@compiletime function createEvasionAbility()
+    new AbilityDefinitionEvasion(ABILITY_ANABOLIC_EVASION)
+        ..setChancetoEvade(1, 0.10)
+        ..setName("Anabolic Evasion")
+        ..setEditorSuffix("(Wurst)")
+
+@compiletime function createSpellShieldAbility()
+    new AbilityDefinition(ABILITY_SPELL_SHIELD, 'ANss')
+        ..setCooldown(1, 75.0)
+        ..setName("Hydra Fins Spell Shield (Amulet)")
+        ..setEditorSuffix("(Wurst)")
+
+@compiletime function createElkSkinBoots() returns ItemDefinition
+    return createBasicItem(ITEM_ELK_SKIN_BOOTS)
+        ..setAbilities(commaList(
```

Comment:
I think closing and opening parentheses should follow the same structure, i.e. if both opening are one line then both closing should be on one line, either

    ..setAbilities(commaList(
        ABILITY_SPEED_BONUS_60,
        ABILITY_INT_BONUS_2
    ))

or 

    ..setAbilities(
        commaList(
            ABILITY_SPEED_BONUS_60,
            ABILITY_INT_BONUS_2
        )
    )